components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -1404,16 +1412,32 @@ public void revokeAccessTokensInBatch(String[] tokens, boolean isHashedToken) th
                     } else {
                         ps.setString(3, getHashingPersistenceProcessor().getProcessedAccessTokenIdentifier(token));
                     }
+                    ps.setInt(4, IdentityTenantUtil.getLoginTenantId());
```

Comment:
why do we need the tenant id for this method?

Reply:
This was added to fix an issue in the logout flow. However since this method supposed to revoke tokens with the given tokenID/ hash, tenant seems redundant here. Will remove the tenant ID.